test/e2e/features/user/tasks/tasklist-full.e2e.js

```diff
@@ -32,6 +32,24 @@ describe('tasklist custom page', function() {
         var formViewer = element(by.css('.modal .FormViewer'));
         expect(formViewer.isPresent()).toBe(true);
       });
+
+      it('should not be displayed for done tasks', function() {
+        var titleRxp = /done/i;
+        element.all(by.css('.TaskFilters li a')).get(3).click();
```

Comment:
should we add an Id to select the filter you want to click instead of using an index which could change?